codec-http/src/test/java/io/netty/handler/codec/http/HttpRequestDecoderTest.java

```diff
@@ -307,7 +308,7 @@ public void testTooLargeInitialLine() {
         assertTrue(channel.writeInbound(Unpooled.copiedBuffer(requestStr, CharsetUtil.US_ASCII)));
         HttpRequest request = channel.readInbound();
         assertTrue(request.decoderResult().isFailure());
-        assertTrue(request.decoderResult().cause() instanceof TooLongHttpLineException);
+        assertThat(request.decoderResult().cause(), instanceOf(TooLongHttpLineException.class));
```

Comment:
this now use hamcrest matchers, while other changes scattered over few other PRs use assertj - It would be good to settle with single library.

Reply:
I use `assertThat#instanceOf` when I have to check if an instance belongs to something. Same as `assertTrue (msg instanceOf ByteBuf)`.

However, I use `assertThat#isNotInstanceOf` when I have to check if an instance does not belong to something. Same as `assertFalse (msg instanceOf String)`.

Using `assertThat#is(not(instanceOf(...))` is not really readable compared to `assertThat#isNotInstanceOf` in my opinion.

This is the reason to settle with both Hamcrest and AssertJ. Also, both libraries are available at runtime so there is no harm.